### DISCUSSION TITLE
Accept invite not displaying for repo invitation

### DISCUSSION THREAD

**Discussion by janayAl** [community/community#29873] [Other Feature Feedback, Questions, & Ideas]
Hi, I'm trying to accept an invitation to a repo, but it's not showing up at all. When I click on the invitation link it leads me to the repo but there are no options to accept or decline being displayed. Is there a reason for this?

I did a search and the only similar issue others are having is that they can see the accept button but it continuously scrolls down to the button so they're unable to click it. My issue is that my accept button isn't showing at all. I haven't found an issue answered regarding this.

**Comment by radar-blip** [comment]
There seems to be some kind of glitch with that lately. 
I dont know if it will help but you can try contacting GitHub Support here: https://support.github.com/request

You could also file a bug report here: [Bug-Report](https://support.github.com/contact/bug-report)

**Comment by janayAl** [comment]
Thank you.

**Reply by radar-blip** [reply]
Anytime. Hope you get it fixed!

**Reply by detolah** [reply]
I encountered this issue too. Is there no way out?
